plugins/woocommerce-admin/client/launch-your-store/hub/sidebar/components/launch-store-hub.tsx

```diff
@@ -48,6 +48,7 @@ export const LaunchYourStoreHubSidebar: React.FC< SidebarComponentProps > = (
 	const sidebarTitle = (
 		<Button
 			onClick={ () => {
+				window.sessionStorage.setItem( 'lysTaskOpen', 'no' );
```

Comment:
I think this is sufficient, but I'm curious if we can do better. I think we can use the `useEffect` hook to remove the session storage value when the component mounts. This way, we can ensure that the session storage value is always cleared when users return to the LYS. Does this make sense?

Reply:
This works well! I made the change in ed00f30. With this change, it made more sense to rename the storage key to `lysWaiting` too.